config.py

```diff
@@ -8,8 +8,8 @@
 # Application settings
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
-API_DESC = 'An API for serving models'
+API_TITLE = 'Object Detector'
+API_DESC = 'Upload an image to predict the pictured objects and their bounding boxes.'
```

Comment:
This sound a little wordy. Maybe something like:
```suggestion
API_DESC = 'Upload an image to predict objects and their bounding boxes.'
```

Reply:
Good suggestion! thank you